spec/views/homes/_events_view_spec.rb

```diff
@@ -1,11 +1,13 @@
-RSpec.describe "homes/_events.html.erb" do
+RSpec.describe "homes/_events.html.erb", type: :view do
```

Comment:
I think so because these tests are more comprehensive of the information that's expected to be present while the feature spec is ensuring all the smaller pieces are working together to render the page.